core/src/test/java/org/mobilitydata/gtfsvalidator/TestUtils.java

```diff
@@ -0,0 +1,33 @@
+package org.mobilitydata.gtfsvalidator;
+
+import static org.junit.Assert.fail;
+
+import java.io.ByteArrayInputStream;
+import java.io.InputStream;
+import java.nio.charset.StandardCharsets;
+import org.mobilitydata.gtfsvalidator.notice.NoticeContainer;
+import org.mobilitydata.gtfsvalidator.notice.ValidationNotice;
+
+public class TestUtils {
+
+  protected TestUtils() {
+    //        Hiding default constructor to avoid instantiation
+  }
+
+  public static InputStream retrieveInputStream(String path) {
+    return TestUtils.class.getResourceAsStream(path);
+  }
```

Comment:
I think this is only used in one-place and for a file that DNE?  I'd get rid of this helper as such.

Reply:
I added it with the idea of using it to load _tests cases_  files but ended up loading the content using the string version of the files inline as a better alternative. Removing it for now.